.devcontainer/install-dependencies.sh

```diff
@@ -87,7 +87,9 @@ if [ "$DEVCONTAINER" = true ]; then
     BUILDX_DEST=/usr/lib/docker/cli-plugins
 else
     TOOL_DEST=$(git rev-parse --show-toplevel)/hack/tools
+    CROSSPLANE_CONFIG_DIR=$(git rev-parse --show-toplevel)/hack/crossplane/config
```

Comment:
This PR is amazing.

The only thing I'm not sure about is, this directory is created/managed by `task`, and I don't think should need to be pre-created in the devcontainer.

I think it might make more sense to just create that directory and include a dummy `config/default/kustomization.yaml` file, that way the path always exists.

I feel that's a better solution than fixing it in the dockerfile as in theory it should fix it for users running the build outside of a container too, and it keeps the Dockerfile clean(er), without knowledge of our specific build layout for the most part.

Reply:
I sent a PR to fix this.